app/controllers/auth_controller.rb

```diff
@@ -1,6 +1,8 @@
 class AuthController < ApplicationController
   helper :auth
 
+  @@attempts = 0
```

Comment:
but then won't that allow the script to brute force just by trying out different user names. He will get unlimited number of tries to make a script guess the password